app/src/main/java/com/asfoundation/wallet/di/AppModule.kt

```diff
@@ -557,6 +557,17 @@ internal class AppModule {
     }
   }
 
+  @Singleton
+  @Provides
+  @Named("NFTNetwork")
+  fun providesDefaultNFTNetwork(): String {
+    return if (BuildConfig.DEBUG) {
+      "https://rinkeby.infura.io/v3/${BuildConfig.INFURA_API_KEY_RINKEBY}"
```

Comment:
It would be nice to eventually use ropsten instead of rinkeby network to have all the dev app in the same network.

Reply:
We choose Open Sea API to provide the NFTs info and they don't support Ropsten at the moment so we will need to stay with Rinkeby for now